thirdparty/antlr/runtime/CMakeLists.txt

```diff
@@ -1,123 +1,118 @@
-
```

Comment:
This file is a part of thirdparty library. All cmake files in thirdparty should not be processed